daily-rev/daily-rev.js

```diff
@@ -0,0 +1,19 @@
+function dailyRev(input) {
+  return input.reduce((acc, cur) => {
+    const date = new Date(cur.timestamp)
+    const dateString = date.toDateString()
```

Comment:
be careful of timezones. This will give a different response in Oregon compared to New York.

You can use toUTCString() instead